src/app/zeko/compile_simple/fake/compile_simple.ml

```diff
@@ -5,6 +5,10 @@ open Snark_params.Tick
 module Proof = struct
   type t = Proof of Field.t (* hash of vk and public input *)
   [@@deriving yojson]
+
+  let is_real = None
+
+  let of_pickles _ = Proof Field.one
```

Comment:
This is invalid. It should check the proof (and thus it needs the associated vk too).